server/graphql/general/src/mutations/display_settings.rs

```diff
@@ -0,0 +1,75 @@
+use async_graphql::*;
+
+use graphql_core::{standard_graphql_error::validate_auth, ContextExt};
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    display_settings_service,
+};
+#[derive(InputObject)]
+pub struct DisplaySettingsInput {
+    pub custom_logo: Option<String>,
+    pub custom_theme: Option<String>,
+}
+
+#[derive(SimpleObject)]
+pub struct UpdateResult {
+    pub logo: bool,
+    pub theme: bool,
+}
+
+impl From<display_settings_service::UpdateResult> for UpdateResult {
```

Comment:
we've been using from_domain and to_domain rather then from implementation. From implementation only used for very common conversions (database error -> api error mainly). 
At the start I used a lot of `from` implementation but Clemens was very strong against it he said it was indirection, so decided to go with from_domain pattern.

You can't really navigate to `into()` implementation that easily.

Although in the latest sync api error handling I've used auto derived `from` through thiserror attributes, I still think we should continue with from_domian pattern here

Reply:
🙏  nice! thanks for explaining. have updated in 2ad9ab9b7 and it is much neater